### PR TITLE
CFY-6307-Manager-blueprints-inputs-should-not-include-default-package…

### PR DESCRIPTION
**Pull request by limor-gs**
…s-and-urls

### CODE REVIEW

File: components/utils.py [hunk 1]
```diff
@@ -520,6 +521,22 @@ def yum_install(source, service_name):
...
+    local_filepath_list = \
+        [fn for fn in glob.glob(os.path.join(CLOUDIFY_SOURCES_PATH, filename))
+         if not os.path.basename(fn).startswith(SINGLE_TAR_PREFIX)]
+    if not local_filepath_list:
+        ctx.abort_operation("File: {0} does not exist in sources path: {1}".
+                            format(filename, CLOUDIFY_SOURCES_PATH))
+    if len(local_filepath_list) > 1:
+        ctx.abort_operation("More than one file: {0} found in sources path:"
+                            " {1}".format(filename, CLOUDIFY_SOURCES_PATH))
+    local_filepath = ''.join(local_filepath_list)
```
Comment (dusking) [inline components/utils.py]: I think that local_filepath_list[0] is more explicit

File: components/utils.py [hunk 2]
```diff
@@ -1133,8 +1150,14 @@ def _download_resource_and_render(self, source, dest, service_name,
     def _download_source_resource(source, local_resource_path):
         is_url = source.startswith(('http', 'https', 'ftp'))
         filename = get_file_name_from_url(source) if is_url else source
-        local_filepath = os.path.join(CLOUDIFY_SOURCES_PATH, filename)
-        is_manager_package = filename.startswith('cloudify-manager-resources')
+
```
Comment (dusking) [inline components/utils.py]: blank line..

File: components/utils.py [hunk 3]
```diff
@@ -1158,7 +1181,18 @@ def _load_node_props(service_name):
 
     def _get_local_file_path(self, service_name, resource_name):
         base_service_res_dir = self.get_resources_dir(service_name)
-        dest_file_path = os.path.join(base_service_res_dir, resource_name)
+
+        if (resource_name.startswith('cloudify')
```
Comment (dusking) [inline components/utils.py]: please move this complicated condition into a function with explanatory name and comments..

File: components/utils.py [hunk 4]
```diff
@@ -1158,7 +1181,18 @@ def _load_node_props(service_name):
...
+            or resource_name.find('-agent_') != -1) \
```
Comment (tehasdf) [inline components/utils.py]: `'-agent_' not in resource_name` is nicer than using `.find` (with its -1 weirdness)

**Pull request merged**